### ISSUE TITLE
Incorrect point count returned for counter 0 for PointRoi when points are added interactively

### ISSUE THREAD

**Issue by melikovk** [imagej/ImageJ#205]
I get the incorrect number of points returned by *getCount(0)* when points are added interactively. Everything works fine when points are added with *addPoint* method or for other counters. See Python examples below.

```python
from ij import WindowManager
from ij.gui import PointRoi
from time import sleep

roi = PointRoi()
imp = WindowManager.getCurrentImage()
imp.setRoi(roi)
counter = 0
roi.setCounter(counter)
print(roi.getCount(counter), len(roi.getContainedPoints()))
for i in range(3):
	roi.addPoint(i*20,i*20)
	print(roi.getCount(counter), len(roi.getContainedPoints()))	
for i in range(5):
	sleep(1)
	print(roi.getCount(counter), len(roi.getContainedPoints()))
```
outputs
```
(0, 0)
(1, 1)
(2, 2)
(3, 3)
(5, 4)
(6, 5)
(7, 6)
(8, 7)
(8, 7)
```
Above I was adding points interactively during the second **for** loop. When counter value is changed to any other value (5 for the output below) everything works fine.
```
(0, 0)
(1, 1)
(2, 2)
(3, 3)
(4, 4)
(5, 5)
(6, 6)
(6, 6)
(7, 7)
```
Thank you for your help

**Comment by scuniff** [comment]
I can re-produce this issue with multi-stack Images, like:

File->Open Samples->MRI- Stack
File->Open Samples->Mitosis (5D Stack)
File->Open Samples->FluorescentCells.tif

Other non Stack images do not seem to have the issue.


See attached screen recordings.

Do you see the same on your Imagej installment?


Screen recording showing problem with Image Stack:
https://user-images.githubusercontent.com/40019379/229549943-5fb6f921-8cff-42ee-a06c-72647e007aa3.mp4



Screen recording showing no problem with non stack Image:
https://user-images.githubusercontent.com/40019379/229550218-12a309da-91a9-4c5d-9dbf-9c1dab047469.mp4

**Comment by melikovk** [comment]
Yes, the issue is with multi-stack images only. I did not realize it during the initial submission, but I see the same as you on my ImageJ installation.

**Comment by melikovk** [comment]
Thank you for exploring this issue. Do you understand why it appears only on multi-stack images as you have discovered before? I also noticed that it appears only in a multi-point mode.

**Comment by rasband** [comment]
This bug is fixed in the ImageJ 1.54f23 daily build. The commit is at https://github.com/imagej/ImageJ/commit/044bd650dcc4fb9943ee3c39980e6610c813317a.
